slsim/lens_pop.py

```diff
@@ -32,6 +35,11 @@ def __init__(
         :type kwargs_deflector_cut: dict
         :param kwargs_source_cut: cuts on the source to be excluded in the sample
         :type kwargs_source_cut: dict
+        :param kwargs_quasars: a dict of the form kwargs_quasars = {'number': 50000,
+         'z_min': 0.1, 'z_max': 5, 'm_min': 17, 'm_max': 23} and param are explained
```

Comment:
I would refer here simply to the specific class where these keyword arguments are used in the init argument as proper arguments